docs/contents.md

```diff
@@ -0,0 +1,208 @@
+# Contents of a package
+
+In order to understand what should be included in a Typst package, it helps to
+understand how a package is processed while and after being submitted to [Typst
+Universe][typst_universe]. The following sections will explain what your
+package may or should not contain during different parts of this processing.
+
+The general gist is that, as your package travels from your repository to end
+users machines, various files probably can (and should) be removed. This is
+done to:
+
+1. Keep [this repository][this_repo] small and
+2. Reduce downloads (and disk usage) for end users
+
+
+## Table of contents
+
+1. [Categories of files][a]
+    1. [Required files][files_required]
+    1. [Documentation files][files_docs]
+    1. [Other files][files_other]
+1. [Package development version][b]
+1. [Published package version][c]
+1. [Imported package version][d]
+
+[a]: #categories-of-files
+[files_required]: #required-files
+[files_docs]: #documentation-files
+[files_other]: #other-files
+[b]: #package-development-version-stage-1
+[c]: #published-package-version-stage-2
+[d]: #imported-package-version-stage-3
```

Comment:
We don't really have ToC in other documentation files, and I don't think this file is long enough to deserve one. Can you remove it, please?